ansible/group_vars/all/all

```diff
@@ -24,6 +24,70 @@ controller_vip: 10.60.253.1
 # IP address from which the logging and monitoring server can be accessed
 alaska_monitoring_server: 10.60.253.3
 
+# Network interface names
+ext_name: ilab
+bdn_name: p3-bdn-nogateway
+lln_name: p3-lln
+
+# Swarm SIP: manila mount
+os_manila_mount_os_config_name: mycloud
+
+# Swarm SIP
+os_container_infra_user: fedora
+os_container_infra_cloud: mycloud
+os_container_infra_state: present
+os_container_infra_cluster_name: swarm-sip
+os_container_infra_cluster_template_name: swarm-fa27-fip-off
+os_container_infra_keypair: default
+os_container_infra_master_count: 1
+os_container_infra_node_count: 6
+os_container_infra_external_interface: "{{ ext_name }}"
+os_container_infra_interfaces:
+- "{{ lln_name }}"
+- "{{ bdn_name }}"
```

Comment:
For things that are specific to the SIP swarm cluster, it might make sense to follow the pattern of having a yaml file under config/ that is pulled in via `-e`. A good question to ask is, how would I create a kubernetes cluster without changing anything in group_vars/?